### PR TITLE
BUG: Fixed typo in OTU-Clustering tutorial

### LINKED ISSUE #311: Typo: OTU Clustering Tutorial
The ["Dereplicating" section](https://docs.qiime2.org/2018.4/tutorials/otu-clustering/#dereplicating-a-sampledata-sequences-artifact) has some unescaped code in there:

screen shot 2018-06-04 at 5 37 42 am

### PR DESCRIPTION
**Pull request by turanoo**
Fixes #311

### CODE REVIEW

File: source/tutorials/otu-clustering.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ Dereplicating a ``SampleData[Sequences]`` artifact
 --------------------------------------------------
 
 If you are beginning your analysis with demultiplexed, quality controlled
-sequences, such as those in a `QIIME 1 ``seqs.fna`` file`_ your first step is
+sequences, such as those in a QIIME 1 ``seqs.fna`` file your first step is
```
Comment (thermokarst) [inline source/tutorials/otu-clustering.rst]: This fixes the unescaped code, but it [doesn't fix the link](https://github.com/turanoo/docs/blame/cb3747713dfe1d2ed188e979b8fe0703c0d5c661/source/tutorials/otu-clustering.rst#L189) that this is supposed to point to.

`QIIME 1 ``seqs.fna`` file` is a link to http://qiime.org/documentation/file_formats.html#post-split-libraries-fasta-file-overview

**Comment by turanoo** [comment]
@thermokarst thanks for reviewing. Please look at the new changes I made. Let me know if this is satisfactory.

File: source/tutorials/otu-clustering.rst [hunk 2]
```diff
@@ -52,8 +52,8 @@ Dereplicating a ``SampleData[Sequences]`` artifact
 --------------------------------------------------
 
 If you are beginning your analysis with demultiplexed, quality controlled
-sequences, such as those in a `QIIME 1 ``seqs.fna`` file`_ your first step is
-to import that data into a QIIME 1 artifact. The semantic type used here is
+sequences, such as those in a QIIME 1 ``seqs.fna`` file_ your first step
```
Comment (thermokarst) [inline source/tutorials/otu-clustering.rst]: Sorry, my [first comment](https://github.com/qiime2/docs/pull/330#discussion_r201359710) wasn't very clear --- I think the original intent of this code was for the link to look like this (displaying as HTML for illustration, we would want to use RST in our implementation):

```html
<a href="http://qiime.org/documentation/file_formats.html#post-split-libraries-fasta-file-overview">
    QIIME 1 ``seqs.fna`` file
</a>
```

Comment (turanoo) [reply source/tutorials/otu-clustering.rst]: @thermokarst that makes more sense. I will go ahead and make the changes accordingly.

**Pull request merged**